oeps/processes/oep-0056-architectural-process.rst

```diff
@@ -0,0 +1,218 @@
+
+See `Appendix A: Advisory Process Principles <#Appendix A: Advisory Process Principles>`__  for more details on the spirit and functioning of the Advisory Process.
+
+Advisory Process Artifacts
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The final artifact of the Advisory Process is an Architectural Decision Record document checked in to a repository relevant to the decision.  Prior to that, however, we suggest as a best practice using Github's Issues and Pull Request tooling to conduct the Advisory Process.  The advantage of doing so is that Issue/Pull Request comments remove the burden of recording who gave particular advice, and allow us to have threaded, focused conversations about various aspects of ADR.
+
+Note that this section is GitHub-specific because that's where the Open edX platform's code resides.  The idea of "issues" and "pull requests" is otherwise portable to other version control platforms.
+
+GitHub Issues (Optional)
+************************
+
+We would encourage decision makers to use a GitHub Issue to have initial conversations around the context of a decision, or to write down upcoming/potential decisions. It may be that - at the start of a project - we're not even sure what decisions need to be made.  They may not even reveal themselves until we're in the middle of coding.
+
+An Issue is a great place to have these initial, loose, conversations about a decision space without the burden of creating a full ADR (creating a new document in the right place, authoring the sections of an ADR, creating a branch, committing, creating a PR, etc.)  It may not be necessary for more well-defined or focused decisions.  We want to preserve this as an optional first step, as we recognize that creating a "formal" ADR can feel inexplicably daunting or like a distraction, especially when possible decisions come up in the course of working on an implementation.  The mental burden of breaking "flow state" and creating a new ADR document is often problematic.  We hope jotting some notes in an Issue as a starting point will allow engineers to record their potential decision point and quickly continue their work.
+
+GitHub Pull Requests
+********************
+
+Whether or not a GitHub Issue is used, a GitHub Pull Request will exist for creation/merging of the ADR.  This is the primary medium for the Advisory Process, and where we expect conversations will take place.  By the time a pull request is created, we expect the decision maker will have written down the context of the decision - necessary to orient advisors to the problem space - as well as draft of their potential decision.
+
+Decision Maker's Checklist
```

Comment:
is tihs OEP going to define the checklist? Or is this a moving target or different on a per-repo basis?

Reply:
We need to decide where to put it - it's going to evolve constantly.  Totally open to suggestions on where to codify a list of people and groups who are stakeholders/subject matter experts in different areas.

Reply:
I could see this being a very small repo, with a few templates defined and either a tracker or a pointer at a tracker in the README. We had a discussion about "ownership" in Standup last week, the notes captured were:

We need a big ol “ownership spreadsheet” again, don’t we?

-> Yes, there are services that provide more beautiful versions of this that may be worth checking out.  Also Backstage which is an opensource project that solves some of these problems. 

Perhaps we can find extant tooling that will help us here. Also when OEP-55 merges (Maintainers) and we codify maintainership and how we're going to advertise per-repo who is the maintainer, I would think the maintainer(s) should be tasked with knowing who the stakeholders/SMEs are for their area.

Reply:
Side note - 2U uses Backstage and I imagine a few folks there have some expertise with it.  We may be able to get some expertise to help us document our expertise. ;)